lnbits/settings.py

```diff
@@ -343,15 +350,16 @@ def set_cli_settings(**kwargs):
 
 # set wallet class after settings are loaded
 def set_wallet_class(class_name: Optional[str] = None):
-    backend_wallet_class = class_name or settings.lnbits_backend_wallet_class
-    wallet_class = getattr(wallets_module, backend_wallet_class)
-    global WALLET
-    WALLET = wallet_class()
+    settings.lnbits_backend_wallet_class = (
+        class_name or settings.lnbits_backend_wallet_class
+    )
 
 
-def get_wallet_class():
-    # wallet_class = getattr(wallets_module, settings.lnbits_backend_wallet_class)
-    return WALLET
+def get_wallet_class(wallet: Optional[str] = None):
+    wallets_module = importlib.import_module("lnbits.wallets")
+    if wallet:
+        return getattr(wallets_module, wallet)()
+    return getattr(wallets_module, settings.lnbits_backend_wallet_class)()
```

Comment:
Reviewer: Please check whether this affects any of your code.

Reply:
These changes would make https://github.com/lnbits/lnbits/pull/1729 redundant as the wallet class would be re-instantiated on every call